test/JDBC/expected/temp_catalog_inval-vu-verify.out

```diff
@@ -0,0 +1,79 @@
+
+-- Catalog entries for temp tables are not visible from places like `pg_class` or `information_schema.tables`
+-- So the best way to check whether the cache entries have been updated locally properly is to just ensure that
+-- it's visible in typical use. 
+-- pg_class, pg_type, pg_attribute
+CREATE TABLE #t1(a int)
+go
+
+insert into #t1 values (1)
+go
+~~ROW COUNT: 1~~
+
+
+select * from #t1
+go
+~~START~~
+int
+1
+~~END~~
+
+
+drop table #t1
+go
+
+select * from #t1
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: relation "#t1" does not exist)~~
+
+
+
+-- constraint, depend, index, sequence, attrdefault
+create table #t2(a int identity primary key, b int default 0)
+go
+
+alter table #t2 add c int
+go
+
+ALTER TABLE #t2 ALTER COLUMN a bigint
+go
+
+alter table #t2 alter column b bigint
+go
+
+insert into #t2(c) values (2)
+go
+~~ROW COUNT: 1~~
+
+
+insert into #t2(c) values (4)
+go
+~~ROW COUNT: 1~~
+
+
+insert into #t2(b, c) values (3, 2)
+go
+~~ROW COUNT: 1~~
+
+
+select * from #t2
```

Comment:
is this sufficient to cover all catalogs we put in ENR? pg_depend, pg_shdepend?

Reply:
pg_depend tracks relationships between the objects (such as between table and index). and pg_shdepend will track the relationship between the role and the temp table. so both are already tested here.